tests/test_runner/test_debug_sql.py

```diff
@@ -90,12 +98,13 @@ def test_output_verbose(self):
         ('''SELECT COUNT(*) AS "__count" '''
             '''FROM "test_runner_person" WHERE '''
             '''"test_runner_person"."first_name" = 'subtest-fail';'''),
+        'ERROR: setUpClass (test_runner.test_debug_sql.TestDebugSQL.ErrorClassTest)',
     ]
 
     verbose_expected_outputs = [
+        'runTest (test_runner.test_debug_sql.TestDebugSQL.PassingTest) ... ok',
         'runTest (test_runner.test_debug_sql.TestDebugSQL.FailingTest) ... FAIL',
         'runTest (test_runner.test_debug_sql.TestDebugSQL.ErrorTest) ... ERROR',
-        'runTest (test_runner.test_debug_sql.TestDebugSQL.PassingTest) ... ok',
```

Comment:
This looks unrelated.

Reply:
Yes fair. I was trying to rearrange to match the orders but now I see `PassingTest()` is used in a different order to its definition.